.evergreen/run-tests.sh

```diff
@@ -51,6 +51,11 @@ fi
 if [ "$SSL" != "nossl" ]; then
     export CLIENT_PEM="$DRIVERS_TOOLS/.evergreen/x509gen/client.pem"
     export CA_PEM="$DRIVERS_TOOLS/.evergreen/x509gen/ca.pem"
+
+    if [ -n "$TEST_LOADBALANCER" ]; then
+        export SINGLE_MONGOS_LB_URI="${SINGLE_MONGOS_LB_URI}&tls=true"
```

Comment:
Why can't the `tls=true` be part of the evergreen variable containing the URI instead of having to patch it up here?

Reply:
I cribbed this from the Java driver. I think the underlying issue is that MO does not yet add SSL/TLS parameters to the uri it reports: https://github.com/10gen/mongo-orchestration/pull/287